Cargo.lock

```diff
@@ -534,7 +534,7 @@ dependencies = [
 
 [[package]]
 name = "benchmark-example-node"
-version = "0.2.2"
+version = "0.2.3-rc2"
```

Comment:
Looks like the `Cargo.lock` is not up to date, as the `Cargo.toml` specifies `rc4` already.